data/json/items/ammo/pebble.json

```diff
@@ -0,0 +1,51 @@
+[
+  {
+    "id": "bearing",
+    "copy-from": "pebble",
+    "type": "AMMO",
+    "name": "bearings",
+    "description": "A box of ball bearings, useful as ammunition for slings.",
+    "weight": 10,
+    "price": 500,
+    "material": "steel",
+    "color": "dark_gray",
+    "count": 50,
+    "range": 2,
+    "damage": 6,
+    "pierce": 3,
+    "dispersion": 90
```

Comment:
I did at the start before changing anything, but it looks like it would be better to do so before and after.